libraries/kunena/upload/upload.php

```diff
@@ -285,75 +299,115 @@ public function ajaxUpload(array $options)
 
 			if (strpos($contentType, 'multipart') !== false)
 			{
-				// Older WebKit browsers didn't support multi-part in HTML5.
-				$exception = $this->checkUpload($_FILES['file']);
-
-				if ($exception)
-				{
-					throw $exception;
-				}
-
-				$in = fopen($_FILES['file']['tmp_name'], 'rb');
-			}
-			else
-			{
-				// Multi-part upload.
-				$in = fopen('php://input', 'rb');
```

Comment:
Why is the stream php://input removed.  We been trying to use streams, my suspicion though it is a behavior of php://input, I know there were a few problems.